src/components/app/Home.css

```diff
@@ -121,6 +121,40 @@
   font-size: 16px;
   text-align: center;
   cursor: pointer;
+  background-color: #fff;
+}
+
+.homeSectionButtonPressed {
+  padding: 0.5em;
+  margin: 15px 0;
+  display: block;
+  border: 2px solid var(--blue-50);
+  text-decoration: none;
+  transition: background-color 300ms;
+  font-weight: bold;
+  font-size: 13px;
+  text-align: center;
+  cursor: pointer;
+  background-color: var(--blue-50);
+  color: #fff;
+
+}
+
+.homeSectionActionButtons {
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
I think we can improve the design of the action buttons (so you can use a new class `.homeSectionActionButton` for them)
1. use `flex: 1` so that they use the whole space evenly.
2. remove the left border for all but the first button so that we don't see a double border

Reply:
Yeah, I tried it and this looks quite good like this. What do you think?